src/app/services/UserService.ts

```diff
@@ -24,7 +24,36 @@ export const UserService = createApi({
             ]
           : [{ type: "userPrices", id: "LIST" }],
     }),
+    getStudentsByFilter: builder.query<
+      StudentResponse[],
+      {
+        userId: string;
+        filter?: string;
+      }
+    >({
+      query: ({ userId, filter }) => {
+        const params = new URLSearchParams();
+
+        if (filter) params.append("filter", filter);
+
+        return `/${userId}/students?${params.toString()}`;
```

Comment:
Acá no hace falta que hagamos el `toString()`


```suggestion
        return `/${userId}/students?${params}`;
```